app/views/report_templates/generate.html.erb

```diff
@@ -11,7 +11,16 @@
                 (_('This will generate a report %s. Based on its definition, it can take a long time to process.') % h(@template.name)) +
                 '</p>').html_safe) %>
 
-  <%= text_f f, :generate_at, label: _('Generate at'), label_help: _('Generate report on a given time. Both date and time are required. Please use format yyyy-mm-dd HH:MM +000') %>
+  <div id="report_generate_at"></div>
+  <%= mount_react_component('DateTime', '#report_generate_at', {
```

Comment:
suggestion: I wonder if it would make sense to create an helper for `DateTime` which will already create that `div` wrapper for you, and all you'll need is add the `props`

Reply:
I am thinking about making rails form integration for all react fields (as useful react form fields are arriving already)

Reply:
@Laviro, this [PR](https://github.com/theforeman/foreman/pull/6443) would replace the need of adding a wrapper div.